fuzzers/forkserver_simple/src/main.rs

```diff
@@ -145,15 +145,16 @@ pub fn main() {
         None => [].to_vec(),
     };
 
+    let forkserver = ForkserverExecutorBuilder::new()
+        .target(res.value_of("executable").unwrap().to_string())
+        .args(&args)
+        .debug_child(debug_child)
+        .shmem_provider(&mut shmem_provider)
+        .build(tuple_list!(time_observer, edges_observer))
```

Comment:
I also pass the tuple_list into `build` for the comand executor, but I'm not sure if that's the "right" way (it was just the easy way :D)

Reply:
observer_tuples is a mandatory field. so I think the current impl is good

Reply:
`program` is also mandatory, not sure if that's a good argument